native_client/javascript/client.js

```diff
@@ -60,12 +59,16 @@ console.error('Loaded model in %ds.', totalTime(model_load_end));
 
 var desired_sample_rate = model.sampleRate();
 
-if (args['lm'] && args['trie']) {
-  console.error('Loading language model from files %s %s', args['lm'], args['trie']);
-  const lm_load_start = process.hrtime();
-  model.enableDecoderWithLM(args['lm'], args['trie'], args['lm_alpha'], args['lm_beta']);
-  const lm_load_end = process.hrtime(lm_load_start);
-  console.error('Loaded language model in %ds.', totalTime(lm_load_end));
+if (args['scorer']) {
```

Comment:
nit: sorry to bother, we'll have to update Sphinx line-references (I wish we had a better solution)